src/main.rs

```diff
@@ -10,7 +10,7 @@ use hyper::Server;
 #[tokio::main]
 async fn main() {
     Server::bind(&SocketAddr::new(IpAddr::V4(Ipv4Addr::UNSPECIFIED), 8080))
-        .serve(app::Builder::new().build())
+        .serve(app::Builder::new().build().unwrap())
```

Comment:
This should not panic on invalid config, log the error and exit with an error code